Application/Features/WorkplaceFeature/Queries/GetWorkplaceListQuery.cs

```diff
@@ -48,6 +54,31 @@ public async Task<GetWorkplaceListQueryResponse> Handle(GetWorkplaceListQueryReq
                         ).Select(x => x.WorkplaceId).Distinct();
                 workplaces = workplaces.Where(x => !busyWorkplacesInThisPeriod.Contains(x.Id));
             }
+
+            if (query.HasWindow is not null)
+            {
+                workplaces = workplaces.Where(x => x.NextToWindow == query.HasWindow);
+            }
+            if (query.HasPc is not null)
+            {
+                workplaces = workplaces.Where(x => x.HasPC == query.HasPc);
+            }
+            if (query.HasMonitor is not null)
+            {
+                workplaces = workplaces.Where(x => x.HasMonitor == query.HasMonitor);
+            }
+            if (query.HasKeyboard is not null)
+            {
+                workplaces = workplaces.Where(x => x.HasKeyboard == query.HasKeyboard);
+            }
+            if (query.HasMouse is not null)
+            {
+                workplaces = workplaces.Where(x => x.HasMouse == query.HasMouse);
+            }
+            if (query.HasHeadset is not null)
+            {
+                workplaces = workplaces.Where(x => x.HasHeadset == query.HasHeadset);
+            }
```

Comment:
It is a good practice to leave empty line after '}'.
Exception is body endings of class, between if and else

Reply:
done